src/test/java/com/vinylteam/vinyl/util/impl/CloneNlParserITest.java

```diff
@@ -38,7 +38,31 @@ void getAllPagesByGenreTest() {
         String genreLink = "https://clone.nl/all/genre/Dubstep?sort=datum&order=desc";
         var allGenrePages = parser.getAllPagesByGenre(genreLink);
         assertFalse(allGenrePages.isEmpty());
-        assertTrue(allGenrePages.size() >= 1);
+    }
+
+    @Test
+    @DisplayName("Checks whether RawOffer is received from HTML Element that represents one vinyl Item")
+    void getRawOffersFromAllOfferLinksTest() {
+        var rawOffer = parser.getRawOfferFromOfferLink("https://clone.nl/item36449.html");
+        assertNotNull(rawOffer);
+        assertEquals("The Paul Breitner EP", rawOffer.getRelease());
+        assertEquals("https://clone.nl/item36449.html", rawOffer.getOfferLink());
+        assertEquals("https://clone.nl/platen/artwork/large/plaatimage36037.jpg", rawOffer.getImageLink());
+        assertEquals(Currency.EUR, rawOffer.getCurrency().get());
+        assertEquals(11.49d, rawOffer.getPrice());
+        assertEquals("Various Artists", rawOffer.getArtist());
+        assertTrue(rawOffer.getGenre().contains("House"));
```

Comment:
Link content is going to change eventually. It's better to check that values in rawOffer after parsing this link aren't default: like price > 0d, currency.isPresent...